cmd/antrea-agent/agent.go

```diff
@@ -63,7 +63,7 @@ const informerDefaultResync = 12 * time.Hour
 func run(o *Options) error {
 	klog.Infof("Starting Antrea agent (version %s)", version.GetFullVersion())
 	// Create K8s Clientset, CRD Clientset and SharedInformerFactory for the given config.
-	k8sClient, _, crdClient, err := k8s.CreateClients(o.config.ClientConnection)
+	k8sClient, _, crdClient, err := k8s.CreateClients(o.config.ClientConnection, o.config.MasterOverride)
```

Comment:
Please use an inclusive name here instead.

Reply:
What is inclusive name?

Reply:
https://github.com/vmware-tanzu/antrea/issues/1675

Reply:
We are avoiding terms like "master". Upstream K8s is also working on avoiding harmful language. I would recommend something like `kubeAPIServerOverride` (adding the `kube` since the Antrea Controller also runs an instance of an API server that each agent connects to).

Reply:
Or just name it kubeAPIServerEndpoint (kubeAPIServerOverride works for me too).

Reply:
+1. Did rename it to `kubeAPIServerOverride`.